app/routers/sessions.py

```diff
@@ -27,13 +34,18 @@ async def create_session(session: Session):
     session_answers = []
     if previous_session is None:
         session["is_first"] = True
+        if quiz["time_limit"] is not None:
+            session["time_remaining"] = quiz["time_limit"]["max"]
 
-        # since we know that there is going to be only one question set for now
         if "question_sets" in quiz and quiz["question_sets"]:
-            question_set_id = quiz["question_sets"][0]["_id"]
-            questions = client.quiz.questions.find(
-                {"question_set_id": question_set_id}, sort=[("_id", pymongo.ASCENDING)]
-            )
+            question_set_ids = [qset["_id"] for qset in quiz["question_sets"]]
+            questions = []
+            for qset_id in question_set_ids:
```

Comment:
This doesn't belong here, no? Why is this in this PR?

Reply:
removed it from current PR